package.json

```diff
@@ -176,6 +176,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.0",
+    "fetch-mock": "^9.1.0",
```

Comment:
We are also using this dependency `"jest-fetch-mock": "^3.0.3"` in the project already 👀

Reply:
I just did a little digging, into both. It seems that it's easier to mock specific URL fetches in `fetch-mock` than in `jest-fetch-mock`.

Reply:
@thekp I've created an issue for this: https://github.com/bbc/simorgh/issues/5960

Reply:
Thanks Sareh 💯